### ISSUE TITLE
Graphql tagged templates with multiple elements are getting newlines appended

### ISSUE THREAD

**Issue by qfox** [prettier/prettier#4615]
**Prettier 1.13.3**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBzATgQwA4AsCOANgAYA6UABBQGZaoC2CMFAHhdBQJ4XDlVUAvPhQC+5YbUwMmFTO0oAjHsKphhYqMQDc5EABoQEbDACW0AM7JQmdOggB3AAo2EllJgBuEEwBN9IBSwwAGs4GABlbEwwEyhUZBh0AFc4A1jzOHQYRzp6TGRqTEIMgwArcxYAISDQiMxGABlYuAKikpAo9Az0ZADMBU5CaH9sdFiYAHVfGFxkAA4ABgNRiAyJrGxe0bhujxaDdDh8JJNDnKk81uLUkAz6EwTkm-NY1EI4AEUkiHgr9ph+lMfDNkAAmAyJTAmQivADCEHolxQUGg+xASQyABV+m5CtcRCIgA)
```sh
--parser babylon
```

**Input:**
```jsx
graphql`
  fragment x on y {
    z
  }

  fragment a on b {
    c
  }
`;

```

**Output:**
```jsx
graphql`
  fragment x on y {
    z
  }

  fragment a on b {
    c
  }

`;

```

**Comment by vjeux** [comment]
This is blocking us from releasing 1.13 at Facebook. Likely caused by #4512. Would be awesome to get a fix and ship a new release.

**Comment by vjeux** [comment]
This is also a regression from 1.12.1

**Comment by ikatyang** [comment]
cc @duailibe @azz @suchipi for the release

**Comment by azz** [comment]
@ikatyang I added you to npm so you can push the patch release.

**Comment by ikatyang** [comment]
Thanks, just published 1.13.4.

**Comment by rubennorte** [comment]
Thanks so much for the quick fix & release!
